src/Http/Controllers/GoogleAnalyticsController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Tightenco\NovaGoogleAnalytics\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Arr;
+use Tightenco\NovaGoogleAnalytics\File;
+use Illuminate\Routing\Controller;
+use Spatie\Analytics\Analytics;
+use Spatie\Analytics\Period;
+
+class GoogleAnalyticsController extends Controller
+{
+    public function index(Request $request)
+    {
```

Comment:
For any of these variables that are used only once, could you just put them inline?
Thanks!